deltacat/utils/pyarrow.py

```diff
@@ -253,6 +254,41 @@ def s3_file_to_table(
         f"Reading {s3_url} to PyArrow. Content type: {content_type}. "
         f"Encoding: {content_encoding}"
     )
+
+    if content_type == ContentType.PARQUET.value:
+        from daft.table import Table
+        from daft import TimeUnit
+        from daft.io import IOConfig, S3Config
+
+        kwargs = {}
+        if pa_read_func_kwargs_provider is not None:
+            kwargs = pa_read_func_kwargs_provider(content_type, kwargs)
+
+        coerce_int96_timestamp_unit = TimeUnit.from_str(
+            kwargs.get("coerce_int96_timestamp_unit", "ms")
+        )
+
+        table = Table.read_parquet(
+            path=s3_url,
+            columns=column_names,
```

Comment:
you may want to use `include_columns` here instead of columns as columns will have all columns while `include_columns` will only have columns to include.

Reply:
done!